hosting/setup-guides/linux.md

```diff
@@ -54,7 +54,7 @@ Remember to check which version to download to ensure it works correctly with yo
 {% endhint %}
 
 1. Download the latest version of `hostd` for your operating system from the [official website](https://sia.tech/software/hostd). For this guide, we'll be downloading the Linux version of `hostd`.
-```bash
+```console
 wget https://github.com/SiaFoundation/hostd/releases/download/v0.2.0/hostd_linux_amd64.zip
```

Comment:
Let’s avoid having versioned download urls. I’ll work with @alexfreska on setting up constant urls for all our software downloads.

Reply:
Ok, it has been updated to use the latest release URL `https://sia.tech/downloads/hostd/latest/linux_amd64`